src/pages/detailsServicePage/DetailsServicePage.tsx

```diff
@@ -68,109 +74,112 @@ export const DetailsServicePage: React.FC = () => {
     keepPreviousData: false,
   })
 
-  const {
-    data: statisticsData,
-    isSuccess: statisticsDataReady,
-    isInitialLoading: statisticsDataLoading,
-  } = useQuery({
-    queryKey: ['serviceStatisticsData', eserviceRecordId],
-    queryFn: () => fetchServiceStatisticsData(eserviceRecordId),
+  const { data: statisticsData, isInitialLoading: statisticsDataLoading } = useQuery({
+    queryKey: ['serviceStatisticsData', eserviceRecordId, mainData?.pollingFrequency],
+    queryFn: () => fetchServiceStatisticsData(eserviceRecordId, mainData?.pollingFrequency),
     onError: (error) => updateSnackbar(true, t('errorRequest', { ns: 'general' }), 'error'),
+    enabled: !!mainData?.pollingFrequency,
   })
 
   return (
     <>
-      {mainDataLoading || probingDataLoading || statisticsDataLoading ? (
-        <DetailsServicePageSkeleton />
-      ) : (
-        <Grid container direction="column" spacing={1} sx={{ height: '100%' }}>
-          {mainDataReady && probingDataReady && statisticsDataReady ? (
-            <>
-              <Grid
-                container
-                item
-                direction="column"
-                sx={{ textAlign: 'center' }}
-                my={5}
-                rowGap={2}
-              >
-                <Grid item>
-                  <Typography variant="h4" component="h1">
-                    {mainData.eserviceName}
-                  </Typography>
-                </Grid>
-                <Grid item>
-                  <Typography variant="body1">{t('subtitle', { ns: 'detailsPage' })}</Typography>
-                </Grid>
+      <Grid container direction="column" spacing={1} sx={{ height: '100%' }}>
```

Comment:
Qui setterei il background color a bianco per seguire il design di Figma
```suggestion
      <Grid container direction="column" spacing={1} sx={{ height: '100%', bgcolor: "white" }}>
```

Reply:
Dato che tutte le pagine presentano uno sfondo bianco, lo imposto nello stile del body